### ISSUE TITLE
Unused packages in renv/library

### ISSUE THREAD

**Issue by robertdj** [rstudio/renv#312]
When I `renv::init()` a project I notice that links are created to *all* packages in `renv`'s cache. 

In my use case I want to later `renv::isolate()` and there it would be nice to only copy the needed packages into  `renv/library` (as specified in`renv.lock`).

I have been searching around, but haven't found a built-in solution. Am I missing anything?

**Comment by kevinushey** [comment]
What is the output of `renv::dependencies()`? Is it possible that your project depends on a package (e.g. `devtools` or `tidyverse`) that requires a large family of packages?

**Comment by robertdj** [comment]
I tried this on a simple project with just one file having the content `library(here)` that uses three packages. 

Weirdly, I first tried this on Linux. This was the first time I used `renv` on that machine and all packages was copied to `renv`'s cache and `renv/library/R-3.6/<platform info folder>` included links to everything in the cache.

Now I'm trying on a Windows machine where I have used `renv` before. Here `renv/library/R-3.6/x86_64-w64-mingw32` contain only links to the relevant package in the cache.
